app/controllers/registrations_controller.rb

```diff
@@ -6,7 +6,7 @@ def after_update_path_for(_resource)
   end
 
   def update_resource(resource, params)
-    if current_user.provider == 'google_oauth2'
+    if current_user.provider == 'google_oauth2' || current_user.provider == 'facebook'
```

Comment:
The conditional for this statement is getting a little too long. You could move `current_user.provider == 'google_oauth2' || current_user.provider == 'facebook'` into it's own private method within this controller.

Here is a great article that details the pattern of refactoring conditionals -> http://designisrefactoring.com/2015/03/09/refactoring-away-a-conditional/

Reply:
Thanks, awesome article! :+1